docs/mixin/alerts/alerts.yaml

```diff
@@ -104,19 +104,36 @@ groups:
   - alert: PromscaleQueryHighErrorRate
     expr: |
       (
-        sum by (job, instance, namespace, type) (
-          rate(promscale_query_requests_total{code=~"5.."}[5m])
+        sum by (job, instance, namespace, type, handler) (
+          rate(promscale_query_requests_total{code=~"5..",handler!="/api/v1/query_range"}[5m])
         )
       /
-        sum by (job, instance, namespace, type) (
-          rate(promscale_query_requests_total[5m])
+        sum by (job, instance, namespace, type, handler) (
+          rate(promscale_query_requests_total{handler!="/api/v1/query_range"}[5m])
         )
       ) > 0.05
     labels:
       severity: warning
     annotations:
       summary: High error rate in querying Promscale.
-      description: "Evaluating queries via Promscale has {{ $value | humanizePercentage }} error rate."
+      description: "Evaluating queries via Promscale {{ $labels.handler }} endpoint has {{ $value | humanizePercentage }} error rate."
+      runbook_url: https://github.com/timescale/promscale/blob/master/docs/runbooks/PromscaleQueryHighErrorRate.md
+  - alert: PromscaleQueryHighErrorRate
```

Comment:
Since this only deals with Range Queries consider reflecting that in the name of the alert.
Maybe `PromscaleQueryRangeHighErrorRate`. Also in the summary.

Reply:
No, we want to have single name for a lot of alerts and differentiate them by labels. This way we can simplify alert routing rules.